docs/pages/workflow/how-expo-works.md

```diff
@@ -113,12 +113,12 @@ The process is essentially the same as opening an Expo app in development, only
 
 ## Opening Expo Apps Offline
 
-The Expo client will automatically cache the most recent version of every app it has opened. When you try to open an Expo app, it will always try and fetch the latest version, but if that fails for whatever reason (including being totally offline) then it will load the most recent cached version.
+the Expo Go app will automatically cache the most recent version of every app it has opened. When you try to open an Expo app, it will always try and fetch the latest version, but if that fails for whatever reason (including being totally offline) then it will load the most recent cached version.
```

Comment:
```suggestion
Expo Go will automatically cache the most recent update of every project it has opened. When you try to open an Expo project, it will always try and fetch the latest update, but if that fails for whatever reason (including being totally offline) then it will load the most recent cached update.
```